clipboard/android/src/main/java/com/capacitorjs/plugins/clipboard/Clipboard.java

```diff
@@ -0,0 +1,75 @@
+package com.capacitorjs.plugins.clipboard;
+
+import android.content.ClipData;
+import android.content.ClipDescription;
+import android.content.ClipboardManager;
+import android.content.Context;
+import com.getcapacitor.JSObject;
+import com.getcapacitor.Logger;
+
+public class Clipboard {
+
+    public enum ContentType {
+        STRING,
+        IMAGE,
+        URL
+    }
+
+    private Context context;
+    private String logTag = Logger.tags(this.getClass().getSimpleName());
```

Comment:
Aren't these usually private, static, final? We should be consistent

Reply:
Not exactly like this. I stole this code from another method. I am adjusting it to how its usually done and so its a static const.